index.html

```diff
@@ -4,7 +4,7 @@
     <meta charset="UTF-8" />
     <link rel="icon" type="image/svg+xml" href="/vite.svg" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Here comes Yamada</title>
+    <title>Here comes nanana</title>
```

Comment:
nananaじゃなくてNaNaNaの方がいいのでは？
```suggestion
    <title>Here comes NaNaNa</title>
```

Reply:
ナトリウムとの関連性がないので微妙かと思いました

Reply:
> nananaじゃなくてNaNaNaの方がいいのでは？

ナトリウムとの関連性がないので微妙かと思いました．